### ISSUE TITLE
Parenthesis the file path prevent hot reloading of content

### ISSUE THREAD

**Issue by weotch** [nuxt/content#975]
### Version
@nuxt/content: 1.14.0
nuxt: 2.15.7

### Steps to reproduce

Create a Nuxt + @nuxt/content project in directory where some ancestor **has parenthesis** in the name.  For example, `~/Work (Open Source)/my-project`

### What is Expected?

When I change text in .md or .yml files within the `content` directory, for the changes to hot reload in my browser when as long as `yarn dev` is running.

### What is actually happening?

Content changes do not hot reload.  I need to restart `yarn dev` before the changes appear in browser.  And they _do_ appear on restart of Nuxt.  This seems to be an issue only with the hot reloading of content feature.  When I move the same project out of a file path containing parenthesis, it works as expected.

**Comment by NozomuIkuta** [comment]
@weotch 

TL;DR

At the moment, you can't enjoy hot-reloading of files in `content` directory, if parentheses are included in ancestor directory names.

---

Spending whole Saturday (I was not so productive 😇), finally I found the root cause:

With parentheses in ancestor directory names, `change` events are not emitted by the file watcher.
Since Nuxt Content hot-reloads UI in the browser, listening the events, it doesn't do anything if there is no event.

Nuxt Content uses [chokidar](https://github.com/paulmillr/chokidar), and the bug is in it.
I already reported the bug to the upstream repository with reproduction demo. 😄 

---

@Atinux @Tahul

Sorry to mention in the weekend.

JFYR, if Nuxt Content v2 and/or Docus depends on chokidar, they would have the same kind of limitation.
I'm glad to write some notes about this limitation somewhere in docs, so let me know if you want.

~~Or, if you take it positively, it might the time to implement your own file watcher like `unjs/watch`.~~
⬆️ Forgive me to say such a stupid thing. My brain is not working since it's almost 4 a.m. in my place.

...I don't know the way to break my habit that I can't sleep, if I find a bug report, until I fix a bug or at least find the root cause.
